packages/react-i18n/src/babel-plugin/shared.ts

```diff
@@ -23,3 +24,28 @@ export function toArrayString(stringArray: string[]) {
     .map(singleStr => JSON.stringify(singleStr))
     .join(', ')}]`;
 }
+
+export function findTranslationBuckets(rootDir) {
+  // `find` is used here instead of Node's glob because it performs much faster
+  // (20s vs 1s in web with ~750 translation folders and 21 langs)
+  const files = execSync(
+    `find ${rootDir} -type d \\( -path ./node_modules -o -path ./build -o -path ./tmp -o -path ./.git -o -path ./public \\) -prune -o -name '*.json' -print | grep /${TRANSLATION_DIRECTORY_NAME}/`,
```

Comment:
time to make one of those famous one line npm libraries that just wraps a unix util :p (jk)